src/chainparams.cpp

```diff
@@ -406,6 +406,15 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_DIP0020].nThresholdMin = 2420; // 60% of 4032
         consensus.vDeployments[Consensus::DEPLOYMENT_DIP0020].nFalloffCoeff = 5; // this corresponds to 10 periods
 
+        // Deployment of decreased proposal fee
+        consensus.vDeployments[Consensus::DEPLOYMENT_GOV_FEE].bit = 7;
+        consensus.vDeployments[Consensus::DEPLOYMENT_GOV_FEE].nStartTime = 1625097600; // July 1st, 2021
+        consensus.vDeployments[Consensus::DEPLOYMENT_GOV_FEE].nTimeout = 1656633600; // July 1st, 2022
```

Comment:
Should bump these to some dates in the future to avoid accidental activation e.g. same date as in https://github.com/dashpay/dash/milestone/28
```suggestion
        consensus.vDeployments[Consensus::DEPLOYMENT_GOV_FEE].nStartTime = 1635292800; // Oct 27th, 2021
        consensus.vDeployments[Consensus::DEPLOYMENT_GOV_FEE].nTimeout = 1666828800; // Oct 27th, 2022
```